src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -134,8 +136,8 @@ export function ContextualSaveBar({
             </div>
           </div>
         </div>
+        {discardConfirmationModalMarkup}
```

Comment:
This has to be moved back out of the ThemeProvider. Otherwise you'll see a dark modal in web